scripts/compile.sh

```diff
@@ -26,6 +26,7 @@ USE_MFEM="Off"
 UPDATE_LIBS=false
 INSTALL_SCALAPACK=false
 MFEM_USE_GSLIB="Off"
+MFEM_USE_LAPACK="NO"
```

Comment:
I'm curious if there are any reasons to have different kinds of options, `"On"` vs `"Off"`, `"Yes"` vs `"No"`, `true` vs `false`. Can we unify them?